components/feedback-form.tsx

```diff
@@ -84,7 +84,9 @@ export const FeedbackForm = () => {
 
   return (
     <div className="mt-8 border-t pt-8">
-      <h1 className="font-bold text-center mb-2">Was this helpful?</h1>
+      <h1 className="font-bold text-center mb-2">
+        Was this helpful? How could we improve it?
+      </h1>
```

Comment:
```suggestion
      <h1 className="font-bold text-center mb-2">
        Was this helpful? What can we improve?
      </h1>
```
Very, very minimal difference, but I think this sounds slightly better